src/website/app/pages/index.js

```diff
@@ -32,23 +32,23 @@ type Pages = Array<Page>;
 
 const pages: Pages = [
   {
-    title: 'Getting Started',
     component: GettingStarted,
+    title: 'Getting Started',
     path: '/getting-started'
```

Comment:
It looks like you moved `title` down to alphabetize, but now it goes c, t, p?

Reply:
yep, good catch